be/src/storage/vectorized/column_aggregate_func.cpp

```diff
@@ -231,8 +231,15 @@ class ReplaceAggregator<ArrayColumn, ArrayState> final : public ValueColumnAggre
     void reset() override { this->data().reset(); }
 
     void aggregate_impl(int row, const ColumnPtr& src) override {
-        this->data().column = src;
-        this->data().row = row;
```

Comment:
Why only array column is special?

Reply:
Good question. I think it is because the child type of the array column is uncertain, and this can reduce the overhead of memory copy in the replace aggregation.